libs/core/src/lib/wizard/public_api.ts

```diff
@@ -0,0 +1,2 @@
+export * from './wizard.module';
+export * from './wizard.component';
```

Comment:
Could you add rest of components/directives from wizard there? I remember that lack of some components caused error during build on external app